pkg/components/components.go

```diff
@@ -742,6 +742,9 @@ func GetClusterRole(allowMultus bool) *rbacv1.ClusterRole {
 					"get",
 					"list",
 					"watch",
+					"create",
+					"update",
```

Comment:
do we need update ? (maybe the reconcile use it when it recreate?)

please consider moving it under the else of `allowMultus`
otherwise there are duplicate / overlapping rbacs due to multusClusterRoles in U/S, which i tried to refrain from

you can move the flag into multusClusterRoles and it will determine if to add * or to add just the diff needed please

Reply:
> do we need update ? (maybe the reconcile use it when it recreate?)

We [do](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/pkg/apply/apply.go#L62).

> 
> please consider moving it under the else of `allowMultus` otherwise there are duplicate / overlapping rbacs due to multusClusterRoles in U/S, which i tried to refrain from
> 
> you can move the flag into multusClusterRoles and it will determine if to add * or to add just the diff needed please

OK